.husky/.secret.patterns

```diff
@@ -0,0 +1,3 @@
+smil\.mil
+(us|aws)-iso
+ic\.gov|sgov\.gov
```

Comment:
Can you explain the use cases of each of these please? For context, no where in our codebase have we had a _.gov_ extension in the past nor do i foresee one popping up in the future

Reply:
Yeah was discussed over slack and opted to get rid of them
It was included in opensource training which is why I had them in the first place